org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServerWrapper.java

```diff
@@ -1040,6 +1041,12 @@ void registerCapability(RegistrationParams params) {
 				serverCapabilities.setTypeHierarchyProvider(Boolean.TRUE);
 				addRegistration(reg, () -> serverCapabilities.setTypeHierarchyProvider(typeHierarchyBeforeRegistration));
 				break;
+			case "textDocument/onTypeFormatting": //$NON-NLS-1$
+				final var onTypeFormattingBeforeRegistration = serverCapabilities.getDocumentOnTypeFormattingProvider();
```

Comment:
given that we are using var to avoid the types, and o to have a short name for an object, I think the variables does not give much value, what about removing them and using
```
				serverCapabilities.setDocumentOnTypeFormattingProvider(reg.getRegisterOptions() instanceof DocumentOnTypeFormattingOptions opts ? opts : null);
				addRegistration(reg, () -> serverCapabilities.setDocumentOnTypeFormattingProvider(serverCapabilities.getDocumentOnTypeFormattingProvider()));
```
?